src/native/libs/System.Native/CMakeLists.txt

```diff
@@ -4,12 +4,6 @@ if (NOT CLR_CMAKE_TARGET_MACCATALYST AND NOT CLR_CMAKE_TARGET_IOS AND NOT CLR_CM
     add_definitions(-DHAS_CONSOLE_SIGNALS)
 endif ()
 
-if (CLR_CMAKE_TARGET_LINUX_MUSL)
-    # Fix up the thread stack size for MUSL to more reasonable size.
-    # TODO: https://github.com/dotnet/runtimelab/issues/791
-    add_definitions(-DENSURE_PRIMARY_STACK_SIZE)
-endif ()
```

Comment:
we have the same in mono, I'm not sure I understand how this is fixed for everything but nativeaot?

Reply:
The issue was specific to native AOT.

We have the same `ENSURE_PRIMARY_STACK_SIZE` in CoreCLR too, however it is not straightforward to get rid of it there since CoreCLR allows configuring the stack size using env variable.

Reply:
Deleting the TODO in #92597